app/grandchallenge/reader_studies/views.py

```diff
@@ -939,7 +988,15 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ReaderStudyViewSet(ReadOnlyModelViewSet):
+class DeleteGroundTruthPermission(DjangoObjectPermissions):
+    def get_required_permissions(self, method, model):
+        return [f"{model._meta.app_label}.change_{model._meta.model_name}"]
```

Comment:
This is still using a change permission?

Reply:
Ah sorry, this needs to be removed, it is no longer used, the action is no longer part of the viewset.